MaterialLib/SolidModels/Lubby2.cpp

```diff
@@ -210,7 +216,7 @@ void Lubby2<DisplacementDim>::calculateResidualBurgers(
 {
     // calculate stress residual
     res.template segment<KelvinVectorSize>(0).noalias() =
-        stress_curr - 2. * (strain_curr - strain_Kel_curr - strain_Max_curr);
+        (stress_curr - stress_t)/dt - 2./dt * ((strain_curr - strain_t) - (strain_Kel_curr - strain_Kel_t) - (strain_Max_curr - strain_Max_t));
```

Comment:
If you can run clang.format, please apply it here. Otherwise, just leave it as it.

Reply:
My clang.format in the vscode editor seems making the code more ugly, so please pull it first, need time to fix the editor's problem...

Reply:
Just sent a PR about clang-format to your branch, please merge it.

Reply:
nice, already merged...